### ISSUE TITLE
[Readme/Website enhancement] The front page should link to a list of suggested wiki hosts for migrating

### ISSUE THREAD

**Issue by fabiorzfreitas** [KevinPayravi/indie-wiki-buddy#919]
I believe this would be a common scenario, both for *users* and *devs/maintainers*:

For this particular example, I'll speak from an *user's* POV

- While I'm doing some interneting, I stumble upon Indie Wiki Buddy. I've always used a good adblock, so that's the first time I ever learned about Fandom and others' bad practices;
- I want to help with a suggestion about this, to a given wiki's maintainers;
- I proceed to open an issue on GitHub or create a post on Reddit for this;
- In my text, I summarize Indie Wiki Buddy's statements and make my point;
- And that's where I stop, without any practical suggestions.

(I've seen wikimatrix.org, including the Wizard function, but comparing 80+ — or 9, when I tested it myself — is a poor replacement for a few popular suggestions from specialists such as yourselves!).

In other words, it's really clear why wikis should move away from Fandom, and how the addon can handle the redirection (by submissions to the wiki list); but, while fandom is well-known, choosing a new host from scratch might be an unnecessary hassle!
